src/AidSelectionStep/AidSelectionStep.tsx

```diff
@@ -59,12 +59,7 @@ export const AidSelectionStep: FunctionComponent<AidSelectionProps> = ({ schema,
       </div>
       <div className="AidSelectionStep-reserved-space">
         {/* Render next step */}
-        {aidColumn && (
-          <>
-            <Divider />
-            {children({ aidColumn })}
-          </>
-        )}
+        {aidColumn && <>{children({ aidColumn })}</>}
```

Comment:
I think it makes more sense to show the empty values warning here, since it is an effect of selecting an AID.
Then the divider can also be kept here, which is even nicer.